assets/src/edit-story/components/tablist/styles.js

```diff
@@ -167,14 +168,19 @@ export const Badge = styled.div`
   }
 `;
 
+export const ScrollableContent = styled.div`
+  max-height: ${({ maxHeight }) => (maxHeight ? `calc(${maxHeight})` : 'none')};
+  overflow-y: ${({ maxHeight }) => (maxHeight ? 'scroll' : 'hidden')};
+`;
+
 export const TabPanel = styled.div`
   height: 0;
   padding: 0;
   visibility: hidden;
-  overflow-y: scroll;
+  overflow-y: hidden;
   overflow-x: hidden;
   background: ${({ theme }) => theme.colors.bg.primary};
-  transition: 250ms ease-in;
+  transition: all 300ms ease-in;
```

Comment:
Fixes the animation grossness! Adds the `overflow-y: scroll` when the tab is expanded.